release-notes/7.0/preview/api-diff/preview6/Microsoft.AspNetCore.App/7.0-preview6_Microsoft.AspNetCore.Authentication.md

```diff
@@ -0,0 +1,36 @@
+# Microsoft.AspNetCore.Authentication
+
+``` diff
+ namespace Microsoft.AspNetCore.Authentication {
++    public static class AuthenticationConfigurationProviderExtensions {
++        public static IConfiguration GetSchemeConfiguration(this IAuthenticationConfigurationProvider provider, string authenticationScheme);
++    }
+     public class AuthenticationService : IAuthenticationService {
+-        [AsyncStateMachineAttribute(typeof(AuthenticationService.<AuthenticateAsync>d__14))]
+-        public virtual Task<AuthenticateResult> AuthenticateAsync(HttpContext context, string? scheme);
++        [AsyncStateMachineAttribute(typeof(AuthenticationService.<AuthenticateAsync>d__15))]
```

Comment:
Is this attribute worth including in the diff?

Reply:
No. :smile:

Reply:
Strange, but thanks for pointing it out. I explicitly added it to my list of attributes to ignore. It was correctly ignored in runtime, but I'm not sure why it didn't work in aspnetcore.